packages/table-next/src/react/__stories__/react-table.story.tsx

```diff
@@ -0,0 +1,393 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Should we disable eslint on spec files entirely?

Reply:
maybe certain rules. we'd still want eslint for typescript linting since the spec/stories are the best way we current have to validate the typings/props/etc